client/webserver/site/src/css/forms.scss

```diff
@@ -238,7 +238,10 @@ button.form-button {
   @extend .flex-center;
 
   position: fixed;
-  inset: 0;
```

Comment:
Why undoing this?  A linter update recently made us do this, and I don't think we should disable that linter.

Reply:
Not supported on webkit. https://github.com/decred/dcrdex/pull/2333#issuecomment-1546698329

Reply:
Oh, right.  That's kinda messed up that webkit doesn't support `inset.  It is supported in every major browser, including Safari since v14.1: https://caniuse.com/mdn-css_properties_inset
What do we know about the webkit version provided by macdrive?

Reply:
I running Safari ver 15.6.1 but know nothing about the WebKit version provided by Macdrive. I think it will use whatever webkit version it finds. 

To get Webkit Version on macOS run: ` defaults read /System/Library/Frameworks/WebKit.framework/Resources/Info.plist CFBundleShortVersionString
` I got `15609` which corresponds with Safari ver 15.6.1 according to [ the comment here ](https://stackoverflow.com/questions/4590102/how-to-check-which-safari-version-is-installed-on-mac-programmatically#:~:text=Show%20activity%20on%20this%20post,%2FWebkit%2FResource%2Finfo.)